### DISCUSSION TITLE
Python package usage

### DISCUSSION THREAD

**Discussion by matthtan** [coqui-ai/TTS#568] [General Q&A]
Hi all,

Apologies if this is written somewhere, I searched and all I could find are the colab notebooks which I couldn't get to run.  I tried the Glow-TTS MultibandMelGAN notebook with those models, after pip installing TTS and the dependencies.  Copying the approrpriate code from the notebook, I get a "cannot import name 'load_config' from TTS.utils.io.  Have I done something wrong?

If the way the notebook has it is not right, is there some resource that shows how to use the TTS Python package from a python script?  I just need to synthesize speech.

Thank you!

**Comment by matthtan** [comment]
Follow on question: I used to use the glow-tts model in version .13 (I had a python script that worked in version .13) using the TTS.utils.synthesizer, which seems to work in the current version.  Is this the preferred way to synthesize speech from a python file?

Also, that glow-tts model no long works because it uses phonemes, but we liked it because we could vary the rate of speech and, coupled with the multiband melgan model, was extremely fast on CPU.  Is there another model that can vary the rate of speech and is extremely fast?

Thanks!

**Reply by erogol** [reply]
Pls check the README. There listed some ways to synthesize speech.

there are also some samples here https://github.com/coqui-ai/TTS/wiki/%F0%9F%90%B8-TTS-Notebooks,-Examples-and-Tutorials

You can try the torchhub sample above. 

The rest might be deprecated but shouldn't be too bad to debug.
